gulpfile.js

```diff
@@ -13,6 +13,14 @@ var lambdaOptions = {
     region: 'eu-west-1'
 };
 
+var env = (function() {
+  function prod(str) { return /prod/gi.test(str); }
+
+  return process.argv.some(prod) ? 'PROD' : 'CODE';
+})();
+
+var Config = require('./email-signup-config')[env];
```

Comment:
This will fail because in order to get the credentials file you need to run the `downloadCredentials` task.
I'm not sure how to get around this other than putting `Config` inside of each task.